examples/GUI/GUIGlobe.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Generated On: 2015-10-5
+ * Class: GUIGlobe
+ * Description: Classe pour créer un menu.
+ */
+
+
+function GUIGlobe() {
```

Comment:
It's not only to display layers, it's also used to create section in the menu. See the "addGUI" function which is used to create the 'RealisticLighting' section.
This class might be used to do other things in the future, so I'm not sure that "layerGUI" is a good name...